static/js/popovers.js

```diff
@@ -1393,7 +1393,6 @@ export function hide_all_except_sidebars() {
     stream_popover.hide_starred_messages_popover();
     hide_user_sidebar_popover();
     hide_mobile_message_buttons_popover();
-    hide_user_profile();
```

Comment:
Can you explain a bit more clearly, either here or in the commit message, why it's already hidden when this function is called?  There's like 6 call points.

Reply:
Explained in the commit message and reordered the commit order to make it more clear

Reply:
Yeah, the critical detail that I was missing is that "hide_user_profile hid a thing that is a modal, not a popover, and the popover logic doesn't make sense for it".